components/eamxx/src/physics/rrtmgp/scream_rrtmgp_interface.cpp

```diff
@@ -222,12 +222,17 @@ namespace scream {
                 real3d &cld_tau_lw_gpt,
                 real2d &sw_flux_up, real2d &sw_flux_dn, real2d &sw_flux_dn_dir,
                 real2d &lw_flux_up, real2d &lw_flux_dn,
+                real2d &sw_clnclrsky_flux_up, real2d &sw_clnclrsky_flux_dn, real2d &sw_clnclrsky_flux_dn_dir,
```

Comment:
This is probably not worth changing (especially since I want for us to refactor this soon to generalize to N-diag calls), but I think it might have been better to order the args to these calls as allsky, clrsky, clnsky, clnclrsky just to keep the "always on" fields first in the list (the cln versions are only computed if the extra bool args are true).